internal/biz/model/common.go

```diff
@@ -334,9 +333,7 @@ type ConsoleHref string
 
 func NewConsoleHref(href string) (ConsoleHref, error) {
```

Comment:
I'm not sure if we want separate types for the different Hrefs? Fundamentally, they are both the same type of thing: URIs. I don't think the console vs API URIs have intrinsically any different invariants. It's like wall color vs ceiling color. They are both colors, just used in different contexts.

Reply:
done.